blockchain/miner.py

```diff
@@ -23,7 +23,8 @@ def proof_of_work(last_proof):
 
     print("Searching for next proof")
     proof = 0
-    #  TODO: Your code here
+    while valid_proof(last_proof, proof) is False:
```

Comment:
`valid_proof(last_hash, proof) ` Valid_proof takes in a hashed value. You are just giving it a number.